scripts/README.md

```diff
@@ -29,14 +29,17 @@ op run -- Rscript sync-data-files.R
 
 The following scripts can be used to generate figures and tables:
 
-1. `fig_1a-sample_disease_barchart.R`: This script is used to generate Figure 1A, which includes a summary of the types of diagnoses found in the ScPCA portal.
+1. `fig-1a_sample-disease-barchart.R`: This script is used to generate Figure 1A, which includes a summary of the types of diagnoses found in the ScPCA portal.
```

Comment:
Do you want to do this as `fig1a_sample-disease-barchart.R` (no `-` after `fig`) to be more consistent with the names you give to the figures themselves?

Reply:
Good point, I'll make that change!